source/mainnet/net/mobile-wallet/update-baker-mw.rst

```diff
@@ -0,0 +1,132 @@
+.. _update-baker-mw:
+
+=====================================
+Change baker options in Mobile Wallet
+=====================================
+
+#. Go to **Accounts**. Tap on the balance area of the baker account you want update or tap **More** |moredetails|.
+
+#. In the hamburger menu |hamburger| tap **Baking**.
+
+#. In the Baker status screen tap **Update baker settings**.
+
+    .. image:: ../images/mobile-wallet/baker-status-mw.png
+
+You have four options:
+
+    - Update baker stake
+    - Update pool settings
+    - Update baker keys
+    - Stop baking
+
+.. image:: ../images/mobile-wallet/baker-hamburger-menu.png
+
+Update baker stake
+==================
+
+If you choose **Update baker stake** you see your balance and the current Baker stake amount. Enter the amount you want to stake. This is the full amount to stake and is not added to the existing stake. You can also choose to adjust your restake setting between **Yes, restake** rewards or **No, don’t restake** rewards. Tap **Continue**.
+
+.. image:: ../images/mobile-wallet/update-baker-stake-amt-mw.png
+
+On the overview screen, check the information. Once you are satisfied, tap **Submit baker transaction**.
+
+Once the transaction is submitted you see a confirmation screen. Tap **Finish** to complete the action.
+
+.. Note::
+
+   Reducing your stake results in a longer :ref:`cool-down period<glossary-cool-down-period>`. The staked amount will be locked during this period and cannot be increased or decreased; you can still change restaking preferences during cool-down. The stake reduction is not effective until the next :ref:`pay day<glossary-pay-day>` after cool-down period ends. During the cool-down period the staked amount continues earning rewards.
+
+Update pool settings
+====================
+
+If you choose **Update pool settings**, you have three options:
+
+.. image:: ../images/mobile-wallet/update-baker-pool-mw.png
+
+- Open pool: open a pool for a previously closed baker
+- Closed for new: close the pool to new delegators. Existing delegators are not affected. You might do this when, for example, the pool is close to meeting one or both of the :ref:`bounding caps<delegation-concept>`. **It is the baker’s responsibility to monitor the stake to make pool management decisions.**
+- Close pool: close a pool permanently.
+
+If you choose **Open pool**:
+
+#. You can optionally enter a URL with information about your baker to give delegators more information about your baker pool to help them research baker pools. This information is not shared in the case of a closed baker.
+
+#. In the Update pool settings overview screen review the information then tap **Submit delegation transaction**.
+
+#. In the Mobile Wallet, you can see that the transaction has been submitted to the chain. Tap **Finish**. Once the transaction is approved, you see the active baker screen where you have the possibility to update baker settings.
+
```

Comment:
```suggestion
#. In the Mobile Wallet, you can see that the transaction has been submitted to the chain. Tap **Finish**. Once the transaction is approved, you can see the active baker screen where you have the possibility to update baker settings.

```
I'm not sure if I understand the sentence, but "you see" is strange.
This "phrase" is repeated many places though :thinking:

Reply:
Kind of the same as with future tense. I will reword the sentence to keep it in present tense without the word "see" in it.